app/backend/src/clj/teet/integration/integration_email.clj

```diff
@@ -69,18 +69,21 @@
     (update msg :subject #(str prefix " " %))
     msg))
 
+(defn- get-msg-type [msg]
+   (:type (first (:body msg))))
+
 (defn- with-data-protection-footer
   "Add data protection clause to text parts"
   [{addr :contact-address} msg]
   (update
-   msg :parts
+   msg :body
    (fn [parts]
      (mapv
       (fn [part]
-        (let [type (get-in part [:headers "Content-Type"])]
+        (let [type (get-msg-type msg)]
```

Comment:
You are processing part but always getting the type from the first part? Shouldn't the type be taken from the current part (just destructure in the `(fn [part] ...`)